clirr-ignores.xml

```diff
@@ -147,4 +147,11 @@
         <justification>False positive, the enclosing class is package-private so this was never exposed</justification>
     </difference>
 
+    <difference>
```

Comment:
I've re-added the method and deprecated it. It doesn't hurt to keep it and it's the cleanest/safest way.